src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -399,6 +399,19 @@
 					ToolTipService.ToolTip="{helpers:ResourceString Name=Install}">
 					<!--  TODO: Add Icon  -->
 				</AppBarButton>
+				
+				<!-- Install Certificate -->
```

Comment:
```suggestion
				<!--  Install Certificate  -->
```
@br1trs XAML Styler must be applied in order to pass the XAML formatting check. You can download it here.
https://marketplace.visualstudio.com/items?itemName=TeamXavalon.XAMLStyler2022